code/game/objects/structures/railing.dm

```diff
@@ -1,6 +1,6 @@
 /obj/structure/railing
 	name = "railing"
-	desc = "A standart steel railing. Prevents from human stupidity."
+	desc = "A standard steel railing. Prevents stupid people from falling into their impending doom."
```

Comment:
This one is still not gramatically correct
An appropriate version would be

"A standard steel railing. Prevents stupid people from falling to their doom."

"into" isnt quite right here, and "impending" has no place here at all